addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-sc-service.js

```diff
@@ -104,6 +104,7 @@ class EnvironmentScService extends Service {
     let envs = await this._scanner({ fields: ['id', 'indexId', 'status', 'outputs'] })
       // Verified with EC2 support team that EC2 describe instances API can take 10K instanceIds without issue
       .limit(10000)
+      .strong()
```

Comment:
Scans have eventual consistency for `read` requests, which would cause it to read outdated records.

More info [here](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_Scan.html)